### PR TITLE
github: add tests workflow

### PR DESCRIPTION
**Pull request by AnnaShaleva**

### CODE REVIEW

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -0,0 +1,192 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+    types: [opened, reopened, synchronize, ready_for_review]
```
Comment (fyrchik) [inline .github/workflows/run_tests.yml:7]: Do we really need `ready_for_review`/`reopened` here?

File: .github/workflows/run_tests.yml [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    paths-ignore:
+      - 'docs/**'
```
Comment (fyrchik) [inline .github/workflows/run_tests.yml:9]: also `**/*.md` ?

File: .circleci/config.yml [hunk 3]
```diff
@@ -69,20 +69,6 @@ jobs:
       - gomod
       - run: go test -v -race ./...
 
-  test_cover:
```
Comment (fyrchik) [inline .circleci/config.yml:72]: Do we need other circle ci actions?

**Review by fyrchik — COMMENTED** [review]
We can also update to go1.16 here and/or drop go1.13.

**Comment by roman-khimov** [comment]
1.16/1.13 actions are planned for the next week, they require some additional attention and we shouldn't touch them here.

File: .github/workflows/run_tests.yml (hunk 1 above)
Comment (AnnaShaleva) [reply .github/workflows/run_tests.yml]: Fixed.

File: .circleci/config.yml (hunk 3 above)
Comment (AnnaShaleva) [reply .circleci/config.yml]: I needed this to ensure that the new Coverage workflow works well and publishes the results to the PR page.

I think that we're not planning to remove circleci actions at the current moment. Let's keep them both for a while, compare and decide which one we'd better get rid of.

**Comment by AnnaShaleva** [comment]
I noticed that circleci's `test_cover` is failing from time to time due to resources issues. And the new `Coverage` works well. On the other hand, the old `test_1_13` and `test_1_14` take ~6-7 minutes to pass vs ~10-11 minutes for the new `Go (1.13)` and `Go (1.14)`.

**Comment by roman-khimov** [comment]
Is f56937e4c1ea85fe00953d1ba838c62b8d2047eb 1.13-specific?

**Comment by AnnaShaleva** [comment]
> Is [f56937e](https://github.com/nspcc-dev/neo-go/commit/f56937e4c1ea85fe00953d1ba838c62b8d2047eb) 1.13-specific?

Yes, 1.13+ tests are able to get `neo-go` dependency and passed successfully.

**Comment by roman-khimov** [comment]
Then let's drop this commit and move on with #1786. This needs to be merged before that, but it's not a big deal to have this particular test failing for a build or two.

**Review by roman-khimov — APPROVED** [review]

**Comment by AnnaShaleva** [comment]
> Then let's drop this commit and move on with #1786. This needs to be merged before that, but it's not a big deal to have this particular test failing for a build or two.

Fixed. And I just found out that we already have github actions enabled in the `dbft` repo, and the same problem is there. I'll fix it in the `dbft` too.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**